pkg/sdk/deps_builder.go

```diff
@@ -31,12 +32,17 @@ func (sdk *poktrollSDK) buildDeps(
 	// Supply the logger
 	deps = depinject.Configs(deps, depinject.Supply(polylog.Ctx(ctx)))
 
+	cosmosClient, err := cosmosclient.NewClientFromNode(config.QueryNodeUrl.String())
```

Comment:
We call it `cometClient` in `NewBlockClient` and `cosmosClient` here.

Can you pick the one correct name and use the same thing in both places?